src/Microsoft.TemplateEngine.Abstractions/TemplateFiltering/MatchInfo.cs

```diff
@@ -44,7 +44,7 @@ public MatchInfo(string name, string? value, MatchKind kind)
         public string? Value { get; }
 
         /// <summary>
-        /// Frequently used filter names. They are also used by <see cref="WellKnownSearchFilters"/>.
+        /// Frequently used filter names. They are also used by Utils.WellKnownSearchFilters.
```

Comment:
Nit: why not cref here?

Reply:
Utils are not accessible in Abstractions, in order to use see cref the member should be resolvable.